training_config/qanet.jsonnet

```diff
@@ -0,0 +1,152 @@
+{
+    "dataset_reader": {
+        "type": "squad",
+        "token_indexers": {
+            "tokens": {
+                "type": "single_id",
+                "lowercase_tokens": true
+            },
+            "token_characters": {
+                "type": "characters",
+                "min_padding_length": 5
+            }
+        },
+        "passage_length_limit": 400,
+        "question_length_limit": 50,
+        "skip_invalid_examples": true
+    },
+    "validation_dataset_reader": {
+        "type": "squad",
+        "token_indexers": {
+            "tokens": {
+                "type": "single_id",
+                "lowercase_tokens": true
+            },
+            "token_characters": {
+                "type": "characters",
+                "min_padding_length": 5
+            }
+        },
+        "passage_length_limit": 1000,
+        "question_length_limit": 100,
+        "skip_invalid_examples": false
+    },
+    "vocabulary": {
+        "min_count": {
+            "token_characters": 200
+        },
+        "pretrained_files": {
+            "tokens": "https://s3-us-west-2.amazonaws.com/yizhongw-dev/glove/glove.840B.300d.lower.zip"
```

Comment:
Do we not have this file in `allennlp/`?  If not, we should add it, and use that link instead.

Reply:
If we're referring to it in an allennlp training config, I'd say we should host the file on our S3 bucket.  Just make sure the name is obvious enough that it's modified by us, and maybe leave a comment in the json file briefly describing what you put here.